src/components/profile/ChangePasswordForm.jsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
```

Comment:
Quick remarks about imports

I usually do this order:
- First external libs (react, redux,...)
- A space
- Internal dependencies sorting by the closest file first (like `./` before `../` before `../../`,...)

Don't know what do you think about it but I like this way